src/main/utils.js

```diff
@@ -20,6 +20,8 @@ export const getPath = directory => {
 }
 
 export const getMenuItem = menuName => {
+  // TODO(fxha): Please use menu id attribute to find menu entries. This will
```

Comment:
Great thanks to you, I did not use `id` attribute before, So If you have time, can you submit a PR for this optimization latter?

Reply:
:+1: